zebra-chain/src/block/difficulty.rs

```diff
@@ -223,12 +223,14 @@ impl CompactDifficulty {
 
     /// Calculate the Work for a compact representation.
     ///
-    /// See `Definition of Work` in the Zcash Specification, and
+    /// See `Definition of Work` in the [Zcash Specification][0], and
     /// `GetBlockProof()` in zcashd.
     ///
     /// Returns None if the corresponding ExpandedDifficulty is None.
     /// Also returns None on Work overflow, which should be impossible on a
     /// valid chain.
+    ///
+    /// [0]: https://zips.z.cash/protocol/canopy.pdf#workdef
```

Comment:
```suggestion
    /// See `Definition of Work` in the [Zcash Specification], and
    /// `GetBlockProof()` in zcashd.
    ///
    /// Returns None if the corresponding ExpandedDifficulty is None.
    /// Also returns None on Work overflow, which should be impossible on a
    /// valid chain.
    ///
    /// [Zcash Specification]: https://zips.z.cash/protocol/canopy.pdf#workdef
```

This should work I think, double check it locally tho

Reply:
I _think_ this will cut out the link text 'Zcash Specification', this is how it currently renders

![image](https://user-images.githubusercontent.com/552961/90186844-dd841b00-dd86-11ea-9b81-40a702b376b9.png)

Reply:
I'm 99% sure its an alias for `[Zcash Specification][Zcash Specification]`, double checking rq

Reply:
yea looks like it works


```rust
/// this is [Pretty Cool]
///
/// [Pretty Cool]: https://github.com/ZcashFoundation/zebra/pull/894#pullrequestreview-467128011
pub fn foo() {}
```

![Screenshot from 2020-08-13 14-07-25](https://user-images.githubusercontent.com/1993852/90187267-5f1b7f00-dd6e-11ea-8a98-7ec15715dc86.png)

Reply:
oh hell yes, 👍👍👍